pyCGM_Single/pycgmStatic.py

```diff
@@ -24,12 +71,65 @@ def rotmat(x=0,y=0,z=0):
     return Rxyz
     
 def getDist(p0, p1):
+    """Get Distance function
+    
+    This function calculates the distance between two 3-D positions.
+
+    Parameters
+    ----------
+    p0 : array
+        Position of first x,y,z coordinate.
+            [ x, y, z ]
+    p1 : array
+        Position of second x,y,z coordinate.
+            [ x, y, z ]
+    Returns
+    -------
+    float 
+        The distance between positions p0 and p1. 
+
+    Example
+    -------
+    >>> from math import *
+    >>> import numpy as np
+
+    >>> p0 = [0,1,2]
+    >>> p1 = [1,2,3]
+    >>> getDist(p0,p1)
+    1.7320508075688772
+
+    >>> p0 = np.array([991.44611381, 741.95103792, 321.35500969])
+    >>> p1 = np.array([117.08710839, 142.23917057, 481.95268411])
+    >>> getDist(p0,p1)
+    1072.3570334681392
+    """
     return sqrt((p0[0] - p1[0])**2 + (p0[1] - p1[1])**2 + (p0[2] - p1[2])**2)
     
 def getStatic(motionData,vsk,flat_foot=False,GCS=None):
-    """
-    Calculate the static offset angle values and return the values in radians
+    """ Get Static Offset function
     
+    Calculate the static offset angle values and return the values in radians
+
+    Parameters
+    ----------
+    motionData : dict
+        Dictionary of marker lists.
+            { [], [], [], ... }
+    vsk : dict, optional
+        Dictionary of various attributes of the skeleton.
+            { [], [], [], ... }
+    flat_foot : boolean, optional
+        A boolean indicating if the feet are flat or not.
+        The default value is False.
+    GCS : array, optional
+        An array containing the Global Coordinate System.
+        If not provided, the default will be set to: [[1, 0, 0], [0, 1, 0], [0, 0, 1]].
+    
+    Returns
+    -------
+    calSM : dict
+        Dictionary containing various marker lists of offsets.
+            { [], [], [], ... }
```

Comment:
Please add an Examples section. Possibly the easiest is to use the helper function to load sample data so you don't need to manually type it.